features/xhr.yml

```
@@ -0,0 +1,56 @@
+name: XMLHttpRequest
+description: The `XMLHttpRequest` API TODO. Also known as XHR.
```

I've restrained myself from including something very opinionated here, like "Use `fetch()` instead."

```suggestion
description: The `XMLHttpRequest` API makes HTTP requests. It's the predecessor to `fetch()`. Also known as XHR.
```